core/src/main/java/feast/core/model/Job.java

```diff
@@ -98,8 +111,8 @@ public boolean isRunning() {
     return getStatus() == JobStatus.RUNNING;
   }
 
-  public String getSinkName() {
-    return store.getName();
+  public boolean isDeployed() {
```

Comment:
Does `isDeployed` mean it is running? Or could have an `id` without actually being live?

Reply:
means it has extId, so it could be running, or could be aborting